lib/fog/dropbox/models/storage/files.rb

```diff
@@ -32,6 +32,7 @@ def head(key, options = {})
 
         def new(attributes = {})
           requires :directory
+          attributes['path'] ||= attributes[:key]
```

Comment:
That was an attempt to get the correct path from the attributes passed via Paperclip.
Paperclip has a fog adapter implementation which passes `:key` for the file path of the file. Not sure if that is non standard or not. 
They work with a number of providers already.

By assigning it to `'path'`,  I was avoiding breaking existing code that may pass in that key as a string.

From the other provider implementations, `:key` seems to be correct. Just didn't want to cause any breaking changes for others.